packages/sanity/src/core/form/inputs/PortableText/PortableTextInput.tsx

```diff
@@ -89,6 +89,7 @@ export function PortableTextInput(props: PortableTextInputProps) {
   const [invalidValue, setInvalidValue] = useState<InvalidValue | null>(null)
   const [isFullscreen, setIsFullscreen] = useState(false)
   const [isActive, setIsActive] = useState(false)
+  const [readOnly, setReadOnly] = useState(readOnlyFromProps)
```

Comment:
Same as earlier re:syncing props with state / local override

Reply:
Good point, I've refactored these to get rid of the `readOnly` state var (use directly from props) and introduced an own state var for the network status to keep these concepts untangled.